src/components/SocialMediaLinks/SocialMediaLinks.test.tsx

```diff
@@ -0,0 +1,476 @@
+import { render, screen } from "@testing-library/react";
+import { axe } from "jest-axe";
+import * as React from "react";
+import renderer from "react-test-renderer";
+import SocialMediaLinks from "./SocialMediaLinks";
+import { socialMediaDataMap } from "./SocialMediaDataMap";
+
+// If you want to see what's happening.
+// screen.debug();
+
+describe("SocialMediaLinks Accessibility", () => {
+  describe("Labels", () => {
+    it("passes axe accessibility test without visible labels", async () => {
+      const { container } = render(<SocialMediaLinks />);
+      expect(await axe(container)).toHaveNoViolations();
+    });
+    it("passes axe accessibility test with labels set to true", async () => {
+      const { container } = render(<SocialMediaLinks showLabels={true} />);
+      expect(await axe(container)).toHaveNoViolations();
+    });
+  });
+  describe("Colors", () => {
+    it("passes axe accessibility test with link color & labels", async () => {
+      const { container } = render(
+        <SocialMediaLinks showLabels={true} color={"link"} />
+      );
+      expect(await axe(container)).toHaveNoViolations();
+    });
+    it("passes axe accessibility test with textInverse color & labels", async () => {
+      const { container } = render(
+        <SocialMediaLinks showLabels={true} color={"textInverse"} />
+      );
+      expect(await axe(container)).toHaveNoViolations();
+    });
+  });
+  describe("Borders", () => {
+    it("passes axe accessibility test with straight borders & labels", async () => {
+      const { container } = render(
+        <SocialMediaLinks showLabels={true} border={"straight"} />
+      );
+      expect(await axe(container)).toHaveNoViolations();
+    });
+    it("passes axe accessibility test with circular borders", async () => {
+      const { container } = render(<SocialMediaLinks border={"circular"} />);
+      expect(await axe(container)).toHaveNoViolations();
+    });
+  });
+});
+
+describe("SocialMediaLinks", () => {
+  it("Renders default", () => {
+    render(<SocialMediaLinks />);
+
+    const links = screen.getAllByRole("link");
+
+    // All the links should be there.
+    // @todo I dunno. Isn't this kinda cheating? It will always be true, so why bother?
+    //   But I am trying to avoid changing the test if/when the socialMediaDataMap gets updated.
+    expect(links).toHaveLength(socialMediaDataMap.length);
+
+    // The links should be what we assigned, above.
+    // @todo Again, I could do this most easily by looping through the links array and mapping
```

Comment:
Agreed and you don't have to test _every_ single case. You can test the first few in the array as a spot check. Same for the label and icon. This is verifying that the component does render to the DOM what we expect and that's the foundational values for the label, url, and icon.